src/App.js

```diff
@@ -4,6 +4,7 @@ import "./App.css";
 import Settings from "./components/settings/Settings";
 import Log from "./components/log/Log";
 import StreamSelect from "./components/streamSelect/StreamSelect";
+import * as Livery from '@exmg/livery';
```

Comment:
It's not a Class but a single module, so I'd call this livery instead of Livery.